apps/homepage/0.10.0/docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 services:
   homepage:
     container_name: ${CONTAINER_NAME}
-    image: ghcr.io/gethomepage/homepage:v0.10.0
+    image: ghcr.io/gethomepage/homepage:v0.10.1
     restart: always
     networks:
       - 1panel-network
```

Comment:
The only change is to upgrade the version of the service's Docker image from `v0.10.0` to `v0.10.1`. There are no other issues indicated.